src/harness/sas_test_harness.py

```diff
@@ -0,0 +1,143 @@
+import ssl
```

Comment:
This looks like the same code from #368 so I will skip reviewing it in this PR.

Reply:
Yes,   used PR 368 code here